src/improvements/tasks/sep/005-opcm-upgrade-v300/VALIDATION.md

```diff
@@ -0,0 +1,402 @@
+                      - Calculate the expected slot for game type 1 using `cast index &lt;KEY_TYPE&gt; &lt;KEY&gt; &lt;SLOT_NUMBER&gt;`:
+                        - `cast index uint32 1 101`
+                      - You should derive a value matching the "Raw Slot" here: 0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f   
+
+  - **Summary:**  Set a new game implementation for game type [CANNON]("https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/dispute/lib/Types.sol#L52")
+  - **Detail:** You can verify this slot corresponds to the game implementation for game type 0 by 
+                     deriving the slot value as follows:
+                     - Notice that [`gameImpls`]("https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol#L57") is a map from a `GameType` to a dispute game address</a>.
+                     - Notice that `GameType` is [equivalent to a ]("https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/dispute/lib/LibUDT.sol#L224-L224") `uint32`.                      
+                     - Notice that the `gameImpls` is [stored at slot 101]("https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/snapshots/storageLayout/DisputeGameFactory.json#L41-L41"). 
+                     - Calculate the expected slot for game type 0 using `cast index &lt;KEY_TYPE&gt; &lt;KEY&gt; &lt;SLOT_NUMBER&gt;`:
+                       - `cast index uint32 0 101`
+                     - You should derive a value matching the "Raw Slot" here: 0xffdfc1249c027f9191656349feb0761381bb32c9f557e01f419fd08754bf5a1b 
+
+  ---
+
+### [`0x16fc5058f25648194471939df75cf27a2fdc48bc`](https://github.com/ethereum-optimism/superchain-registry/blob/08e3fe429c776a532c2b6dc09571fc13e6dba5d4/superchain/configs/sepolia/op.toml#L58)  (OptimismPortal2) - Chain ID: 11155420
+
+- **Key:**          `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+  - **Decoded Kind:**      `address`
+  - **Before:** `0x2D7e764a0D9919e16983a46595CfA81fc34fa7Cd`
+  - **After:** `[0xB443Da3e07052204A02d630a8933dAc05a0d6fB4](https://github.com/ethereum-optimism/superchain-registry/blob/00208555c3c356d6596feedb619da989de478ed7/validation/standard/standard-versions-sepolia.toml#L13-L13)`
+  - **Summary:**           ERC-1967 implementation slot
+  - **Detail:**            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
+  ---
+
+### [`0x1eb2ffc903729a0f03966b917003800b145f56e2`](https://github.com/ethereum-optimism/superchain-registry/blob/b40cf4289c58e28eb1c791f9ad5724380b7516a7/validation/standard/standard-config-roles-sepolia.toml#L3)  (ProxyAdminOwner (GnosisSafe)) - Chain ID: 11155420
```

Comment:
This state diff is missing the approve hash state diffs that Tenderly includes. Can you include them here? Here's an example of me doing it: https://github.com/ethereum-optimism/superchain-ops/pull/848/files#diff-ea05bc2d7951a55d5c82b5f1f3f5c02286e5a677ad7d3a7af1dc1df494ffc8edR96-R128

Reply:
![CleanShot 2025-04-10 at 11 17 51](https://github.com/user-attachments/assets/d03f0299-f28a-494c-99d2-1b40abc0ec51)

Reply:
I addresses this in my latest commit.